src/backend/utils/misc/queryenvironment.c

```diff
@@ -57,6 +57,7 @@ struct QueryEnvironment
 {
 	List	   *namedRelList;
 	List	   *dropped_namedRelList;
+	List 	   *savedCatcacheMessages;
```

Comment:
in general, do we have clean separation of ENR code specific to babelfish and ENR code for PG?